bin/node/runtime/src/lib.rs

```diff
@@ -81,7 +81,7 @@ pub const VERSION: RuntimeVersion = RuntimeVersion {
 	// and set impl_version to 0. If only runtime
 	// implementation changes and behavior does not, then leave spec_version as
 	// is and increment impl_version.
-	spec_version: 222,
+	spec_version: 223,
```

Comment:
I am not sure if this is a spec change.

Reply:
It is though right? Cause I will need to make an update to the Polkadot which relies on this API in the Claims pallet.

I incorrectly commented that I bumped `impl` but I intended to bump spec.